packages/mobile/src/screens/search-screen/content/EmptySearch.tsx

```diff
@@ -1,41 +1,43 @@
-import { StyleSheet, View, Text } from 'react-native'
+import { View } from 'react-native'
 
 import IconBigSearch from 'app/assets/images/iconBigSearch.svg'
-import { useColor, useTheme } from 'app/utils/theme'
+import { Text } from 'app/components/core'
+import { makeStyles } from 'app/styles'
+import { useColor } from 'app/utils/theme'
 
 const messages = {
   empty: 'Search for Artists, Tracks, Playlists, & Albums',
   noResults: 'Sorry, we couldn’t find anything that matches'
 }
 
-const styles = StyleSheet.create({
+const useStyles = makeStyles(({ spacing }) => ({
   emptyContainer: {
     marginTop: 116,
```

Comment:
nit: use `spacing` here too

Reply:
or not, 116 is really big lol

Reply:
yeah with these really random values that existed i kinda throw my hands :D